client/dist/index.html

```diff
@@ -2,7 +2,11 @@
 <html>
 
 <head>
- <title>Sound Board</title>
+ 	<link rel="stylesheet" href="style.css">
+	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
+
+
+	<title>Sound Board</title>
```

Comment:
Just to be picky but it's 'StreamBoard'

Reply:
Lol. True, at least it's not Video Board though.